sdk/iot/pnp/README.md

```diff
@@ -0,0 +1,48 @@
+# Azure IoT Clients for Embedded C: Azure IoT Device Provisioning Client
```

Comment:
I don't believe PNP is in the repo anymore. You might have to rebase this branch